pypeit/scripts/arcid_plot.py

```diff
@@ -44,6 +43,8 @@ def main(args):
 
     from linetools.utils import loadjson
 
+    # TODO: This must be an out-dated script that is never used.
```

Comment:
yup

Reply:
Okay, made a note that gets propagated to the docs, but leaving the script in case someone wants to update it.